chromatic_fitting/models/lightcurve.py

```diff
@@ -607,11 +673,14 @@ def sample(
         self,
         summarize_step_by_step=False,
         summarize_kw={"round_to": 7, "hdi_prob": 0.68, "fmt": "wide"},
+        sampling_method=sample,
+        sampling_kw={"init": "adapt_full"},
```

Comment:
add sampling_method kw - by default it's pymc3.sample but we can change to pymc3_ext.sample if we want